i18nify-data/go/country-subdivisions/data/SG.json

```diff
@@ -3,11 +3,11 @@
   "states": {
     "nan": {
       "name": "nan",
-      "cities": {
-        "Yio Chu Kang": {
+      "cities": [
+        {
           "name": "Yio Chu Kang",
           "timezone": "Asia/Singapore",
-          "region_name/district_name": "North-East Region",
+          "region_name": "nan",
```

Comment:
this is a schema change.. please ensure we are writing a proper release for this to avoid breaking change for the consumers.